.devcontainer/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM ruby:2.6.3-buster
```

Comment:
A pretty standard Dockerfile for a Ruby development container. VS Code will mount the source into the container and expose the ports.